src/services/auth/auth.ts

```diff
@@ -20,46 +19,10 @@ export async function authenticateCredentialLogin(credentials: Credentials) {
 
   const { email, password } = credentials;
   const user = await findByEmail(email);
-  if (!user) {
-    throw new AuthError({ message: 'Provided email or password does not match.' });
-  }
-
-  const isMatchedPassword = await crypt.compareWithHashValue(password, user.password);
+  const isMatchedPassword = await hasUserPasswordMatched(user, password);
```

Comment:
On eon side the variable name uses `is` and on the other the method uses `has`, so it is kind of inconsistent. How about the following:

```suggestion
  const isValidPassword = await validateUserPassword(user, password);
```

Naming things is hard 💦